### PR TITLE
Setup Travis secrets to enable access to Firebase

### PR DESCRIPTION
**Pull request by Matthias-Schaub**
With this Pull Request the Travis instance can now access the Firebase CI (firebase-test) project. The Docker container firebase-deploy now builds and runs successfully and tests as well.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,41 +3,24 @@
...
 
 install:
   - pip install --upgrade pip setuptools
   - pip install flake8 black
 
 script:
   - docker-compose up --build --detach firebase_deploy
+  - docker-compose up --build --detach postgres
   - docker-compose run mapswipe_workers python -m unittest discover --verbose --start-directory tests/unittests/
   #- docker-compose run mapswipe_workers python -m unittest discover --verbose --start-directory tests/integration/
```
Comment (Hagellach37) [inline .travis.yml:25]: I think we should then also uncomment the intregration test

File: docs/source/dev_setup.md [hunk 2]
```diff
@@ -136,13 +136,11 @@ For more information refer to the official [Reference on Cloud Function for Fire
 
 ## Travis Setup
 
-Configuration for travis setup is utilizing environment variables.
+A Travis instance is used to build MapSwipe Workers and run tests.
 
-One difference to production or development setup is that the service account key as json is stored in the environment variables `FIREBASE_CONFIG` as text. To make this work special characters as to be escaped first. This command will simply escape every character:
+For the configuration of Travis environment variables are used: https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings
```
Comment (Hagellach37) [inline docs/source/dev_setup.md:141]: could you add here which environment variables are set in travis. I think we only set some and not all. E.g. `bing_imagery_api_key` or so is not set and doesn't need to be set.

File: mapswipe_workers/tests/unittests/test_firebase_connection.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+class TestFirebaseConnection(unittest.TestCase):
+    def setUp(self):
+        self.fb_db = auth.firebaseDB()
+
+    def tearDown(self):
+        self.fb_db = None
+
+    def test_create_project(self):
+        ref = self.fb_db.reference("/v2/test")
+        ref.push("test")
```
Comment (Hagellach37) [inline mapswipe_workers/tests/unittests/test_firebase_connection.py:15]: I think if you push something here, you should also remove it during tear down?

maybe there is a simpler test to check of the Firebase Connection is working.

File: mapswipe_workers/tests/unittests/test_firebase_connection.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        result = ref.get()
+        self.assertIsNotNone(result)
+        ref.set({})
```
Comment (Hagellach37) [inline mapswipe_workers/tests/unittests/test_firebase_connection.py:18]: ah okay here you delete the data again...

File: mapswipe_workers/tests/unittests/test_firebase_connection.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class TestFirebaseConnection(unittest.TestCase):
+    def setUp(self):
+        self.fb_db = auth.firebaseDB()
+
+    def tearDown(self):
+        self.fb_db = None
+
+    def test_create_project(self):
```
Comment (Hagellach37) [inline mapswipe_workers/tests/unittests/test_firebase_connection.py:13]: we should rename this function. Will this return something? Can we test, assert for a specific error in firebase? e.g. no connection or something like this?

e.g. defined here: https://firebase.google.com/docs/reference/admin/python/firebase_admin.exceptions

File: .gitignore [hunk 6]
```diff
@@ -135,3 +132,5 @@ nginx.conf
 
 .DS_Store
 
+# Custom setup script
+setup-custom.sh
```
Comment (Hagellach37) [inline .gitignore:136]: when is this script used?

File: docs/source/dev_setup.md (hunk 2 above)
Comment (Matthias-Schaub) [reply docs/source/dev_setup.md]: Sure. I added more information on the travis setup to the docs.

File: mapswipe_workers/tests/unittests/test_firebase_connection.py (hunk 3 above)
Comment (Matthias-Schaub) [reply mapswipe_workers/tests/unittests/test_firebase_connection.py]: Yes there is a simpler test indeed. Now the tests just gets the keys at root with .get(shallow=True).

File: mapswipe_workers/tests/unittests/test_firebase_connection.py (hunk 5 above)
Comment (Matthias-Schaub) [reply mapswipe_workers/tests/unittests/test_firebase_connection.py:13]: Good idea. I created an issue for this: #408

File: .gitignore (hunk 6 above)
Comment (Matthias-Schaub) [reply .gitignore:136]: At the moment I use this script to setup my environment for development.

**Pull request merged**